recipes/_tomcat-attributes.rb

```diff
@@ -1,72 +1,77 @@
-node.default['artifacts']['kryo-serializers']['version'] = "0.34"
+node.default['artifacts']['kryo-serializers']['groupId'] = 'de.javakaffee'
+node.default['artifacts']['kryo-serializers']['artifactId'] = 'kryo-serializers'
+node.default['artifacts']['kryo-serializers']['version'] = '0.34'
 node.default['artifacts']['kryo-serializers']['type'] = 'jar'
 node.default['artifacts']['kryo-serializers']['destination'] = "#{node['alfresco']['home']}/lib"
 node.default['artifacts']['kryo-serializers']['owner'] = node['alfresco']['user']
 
-node.default['artifacts']['kryo']['groupId'] = "com.esotericsoftware"
-node.default['artifacts']['kryo']['artifactId'] = "kryo"
-node.default['artifacts']['kryo']['version'] = "3.0.3"
+node.default['artifacts']['kryo']['groupId'] = 'com.esotericsoftware'
+node.default['artifacts']['kryo']['artifactId'] = 'kryo'
+node.default['artifacts']['kryo']['version'] = '3.0.3'
 node.default['artifacts']['kryo']['type'] = 'jar'
 node.default['artifacts']['kryo']['destination'] = "#{node['alfresco']['home']}/lib"
 node.default['artifacts']['kryo']['owner'] = node['alfresco']['user']
 
-node.default['artifacts']['minlog']['groupId'] = "com.esotericsoftware"
-node.default['artifacts']['minlog']['artifactId'] = "minlog"
-node.default['artifacts']['minlog']['version'] = "1.3.0"
+node.default['artifacts']['minlog']['groupId'] = 'com.esotericsoftware'
+node.default['artifacts']['minlog']['artifactId'] = 'minlog'
+node.default['artifacts']['minlog']['version'] = '1.3.0'
 node.default['artifacts']['minlog']['type'] = 'jar'
 node.default['artifacts']['minlog']['destination'] = "#{node['alfresco']['home']}/lib"
 node.default['artifacts']['minlog']['owner'] = node['alfresco']['user']
 
-node.default['artifacts']['reflectasm']['groupId'] = "com.esotericsoftware"
-node.default['artifacts']['reflectasm']['artifactId'] = "reflectasm"
-node.default['artifacts']['reflectasm']['version'] = "1.11.3"
+node.default['artifacts']['reflectasm']['groupId'] = 'com.esotericsoftware'
+node.default['artifacts']['reflectasm']['artifactId'] = 'reflectasm'
+node.default['artifacts']['reflectasm']['version'] = '1.11.3'
 node.default['artifacts']['reflectasm']['type'] = 'jar'
 node.default['artifacts']['reflectasm']['destination'] = "#{node['alfresco']['home']}/lib"
 node.default['artifacts']['reflectasm']['owner'] = node['alfresco']['user']
 
-node.default['artifacts']['asm']['groupId'] = "org.ow2.asm"
-node.default['artifacts']['asm']['artifactId'] = "asm"
-node.default['artifacts']['asm']['version'] = "5.1"
+node.default['artifacts']['asm']['groupId'] = 'org.ow2.asm'
+node.default['artifacts']['asm']['artifactId'] = 'asm'
+node.default['artifacts']['asm']['version'] = '5.1'
+node.default['artifacts']['asm']['type'] = 'jar'
+node.default['artifacts']['asm']['destination'] = "#{node['alfresco']['home']}/lib"
+node.default['artifacts']['asm']['owner'] = node['alfresco']['user']
+
+node.default['artifacts']['asm']['groupId'] = 'org.objenesis'
```

Comment:
this and all the attributes belonging to this jar should be `node.default['artifacts']['objenesis']` instead of `node.default['artifacts']['asm']